python/benchmark/benchmark/base.py

```diff
@@ -87,6 +87,12 @@ def __init__(self, argv: List[Any]) -> None:
             action="store_true",
             help="do not stop spark session when finished",
         )
+        self._parser.add_argument(
+            "--verbose",
+            type=int,
+            default=0,
+            help="verbose param for cuml classes",
```

Comment:
Since the type is int instead of bool, need help information to know what different int values mean to the verbose argument.

Reply:
Changed to boolean and set internal verbose level to max.